book/src/concepts/what-is-a-transaction.md

```diff
@@ -29,6 +29,24 @@ Transactions consist of:
 - a gas object - the `Coin` object used to pay for the transaction;
 - gas price and budget - the cost of the transaction;
 
+## Inputs
+
+Transaction inputs are the arguments for the transaction and are split between 2 types:
+- Pure arguments: These are mostly [primitive types](../move-basics/primitive-types.html) with some
+extra additions. A pure argument can be:
+    - [`bool`](../move-basics/primitive-types.html#booleans).
+    - [integer](../move-basics/primitive-types.html#integers) (`u8`, `u16`, `u32`, `u64`, `u128`, `u256`).
+    - [`address`](../move-basics/address.html).
+    - [`std::string::String`](../move-basics/string.html), UTF8 strings.
+    - [`std::ascii::String`](../move-basics/string.html#ascii-strings), ASCII strings.
+    - [`vector<T>`](../move-basics/vector.html), where `T` is a pure type.
+    - [`std::option::Option<T>`](../move-basics/option.html), where `T` is a pure type.
+    - [`std::object::ID`](../storage/uid-and-id.html), takes part in the Unique IDentifier (UID) of an object. See also [What is an Object](../object/object-model.html).
```

Comment:
I wouldn't mention anything UID here. ID is just a type which typically points to an object, though it's not a requirement.

Reply:
[621aff8](https://github.com/MystenLabs/move-book/pull/91/commits/621aff82c8207ec169158b902bec6771bfa622a5)